btr-web/btr-main-app/cypress/e2e/layouts/breadcrumb.cy.ts

```diff
@@ -27,3 +27,13 @@ describe('Layout -> Breadcrumb', () => {
     cy.get('[data-cy="crumb-link"]').eq(2).contains('0871427 B.C. LTD.')
   })
 })
+
+describe('Layout (person) -> Breadcrumb', () => {
+  it('shows expected breadcrumb information for profile page', () => {
+    cy.visit('/my-registries-details')
+    cy.get('#bcros-breadcrumb')
+    // shows the expected breadcrumbs for owner change
+    cy.get('[data-cy="crumb-link"]').should('have.length', 1)
+    cy.get('[data-cy="crumb-link"]').eq(0).contains('My BC Registries Details')
```

Comment:
Just a thing I learned for future reference.

`.contains` is a selector similar to `.get` and allows you to chain.
while `.should('exist')` is assertion.

I also used `.contains `at first, but than started using `.should('exist')` as assertion when want to convey that I am checking existence of the component/text on the page.

Reply:
I'll update

Reply:
it's more in the message they want to convey I would say. With `.should('exist')` we are explicitly stating we are trying assert that this element/text exists.